README.md

```diff
@@ -1,2 +1,31 @@
-# easy_office_map
-Open source office map app created during Inkovate 2016 - CustomInk's internal hack-a-thon. 
+# Easy Office Map
+
+Believe it or not, a last minute pivot in a hack-a-thon project led to the creation of Easy Office Map. Today the map is widely used within CustomInk to figure out where your next meeting is. We thought other organizations might benefit as well so we have open-sourced the code.
+
+Easy Office Map is a very bare bones application but it's pretty easy to use. The existing sample maps should give you an idea of how to implement it for your office. It basically boils down to replacing the map images with your own and adding room names and coordinates. DEV_MODE makes it very easy.
```

Comment:
It's more then bare bones! "very bare bones application but it's pretty easy to use" It's a "simple solution"

Reply:
Maybe sell "DEV_MODE makes it very easy" as "it comes with an administration interface called DEV_MODE for editing your own maps and making changes"